ocaml/xcp-rrdd/bin/rrdd/rrdd_monitor.ml

```diff
@@ -62,14 +62,49 @@ let merge_new_dss rrd dss =
     )
     rrd new_dss
 
+module OwnerMap = Map.Make (struct
+  type t = ds_owner
+
+  let compare a b =
```

Comment:
Is there anything special about this order? Would the built-in `compare` not yield a similar result? In that case `let compare = compare` with a comment would be sufficient.

Reply:
I'm wary about using the polymorphic compare here due to its overhead compared to a hand-made function